providers/infoblox/infoblox.go

```diff
@@ -14,11 +15,13 @@ import (
 )
 
 const (
-	versionURL   = "/wapi/v1.4/"
-	authURL      = "/wapi/v1.4/zone_auth"
-	recordURL    = "/wapi/v1.4/record"
-	recordTxtURL = "/wapi/v1.4/record:txt"
-	recordAURL   = "/wapi/v1.4/record:a"
+	versionURL   = "/wapi/v1.5/"
+	authURL      = versionURL + "zone_auth"
+	recordURL    = versionURL + "record"
+	recordTxtURL = versionURL + "record:txt"
+	recordAURL   = versionURL + "record:a"
+	maxResults 	 = "1000"
+	paginate 	 = "_return_as_object=1&_max_results=" + maxResults + "&_paging=1"
```

Comment:
@rawmind0 does hardcoded paging=1 mean that we only get the first page?

Reply:
@alena1108 , no, meaning of paging=1 is just to activate pagination answer from infoblox api. We get all pages.

Reply:
@rawmind0 can we rename the parameter? as it's a bit misleading. To something like `firstPage`. That would indicate that further pagination will be handled down in the code

Reply:
No problem. Renaming it.